.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Docker Superhero Wallet image
+
+on:
+  push:
+    branches: [ master, develop ]
```

Comment:
here i guess we could only check for master

Reply:
Done